### DISCUSSION TITLE
Select the row with a click anywhere (not only on the selection checkbox)

### DISCUSSION THREAD

**Discussion by dariogliendo** [tusen-ai/naive-ui#5083] [General]
I would like my rows to be selectable by a click anywhere in it, or at least to have a click handler for the whole rows without having to declare a render method for each cell to handle this behaviour.
Have someone faced this same problem? Any possible solutions? What do you think of adding this as a feature?

**Comment by jahnli** [comment]
`row-props`  + `checked-row-keys`  ？
